contents/BlockamokRemix.oscmeta

```diff
@@ -0,0 +1,34 @@
+{
+    "information": {
+        "name": "Blockamok Remix",
+        "author": "Mode8fx",
+        "category": "games",
+        "supported_platforms": [
+            "wii",
+            "vwii",
+            "wii_mini"
+        ],
+        "peripherals": [
+            "Wii Remote",
+            "Classic Controller",
+            "GameCube Controller",
+            "SDHC"
+        ],
+        "version": "auto"
+    },
+    "source": {
+        "type": "github_release",
+        "format": "zip",
+        "repository": "Mode8fx/blockamok",
+        "file": "BlockamokRemix-wii.zip"
+    },
+    "treatments": [
```

Comment:
You can remove the "treatments" btw. Once you extract the files, it will be just apps/BlockamokRemix

Reply:
Ah, right. Removed.